applications/nrf5340_audio/src/modules/dfu_entry.c

```diff
@@ -46,24 +33,6 @@ static const struct bt_data ad_peer[] = {
 		      0x4c, 0xb7, 0x1d, 0x1d, 0xdc, 0x53, 0x8d),
 };
 
-/* MCUMGR related functions */
-static void mcumgr_register(void)
```

Comment:
It seems that mgmt register function will be called automatically once CONFIG_MCUMGR_XX_MGMT is enabled as well as smp_bt_register in this PR https://github.com/nrfconnect/sdk-zephyr/pull/1025.
What can application do to choose advertise SMP_BT service or not?
In LE audio, we don't want to advertise SMP_BT while normal application is operating.
We'd like device to advertise SMP_BT service if specified button is pressed during bootup.

Reply:
You can advertise the service whenever is needed by setting it in the advert, remote devices will only see it from adverts if it is there, but the service will always be on the device, or you can unregister it by calling the unregister function then re-register it again later